pkg/secrets/clients/conjur/conjur_client.go

```diff
@@ -8,13 +8,13 @@ import (
 )
 
 /*
-	Client for communication with Conjur. In this project it is used only for
-    batch secrets retrieval so we expose only this method of the client.
+Client for communication with Conjur. In this project it is used only for
+batch secrets retrieval so we expose only this method of the client.
 
-	The name ConjurClient also improves readability as Client can be ambiguous.
+The name ConjurClient also improves readability as Client can be ambiguous.
 */
 type ConjurClient interface {
-	RetrieveBatchSecrets([]string) (map[string][]byte, error)
+	RetrieveBatchSecretsSafe([]string) (map[string][]byte, error)
```

Comment:
Are we sure it's totally safe now or are there other corner cases that might sneak in? :)

Reply:
Oh right, you're just using a different `conjur-api-go` method? Nah, that's fine then.